scripts/bundleSizeConfig.js

```diff
@@ -2,5 +2,5 @@ module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
   MIN_SIZE: 647,
-  MAX_SIZE: 706,
+  MAX_SIZE: 720,
```

Comment:
**Not related to this PR**
This bundle size is getting pretty large, we should consider bundle splitting in a separate issue.